### ISSUE TITLE
Add Dev Resources on README

### ISSUE THREAD

**Issue by marcelscruz** [public-apis/public-apis#1716]
Hi.

I recently created a project called [Dev Resources](https://devresourc.es), where I'm using the public-apis list (with due credits) for the ["Public APIs" section](https://devresourc.es/tools-and-utilities/public-apis).

Users are already helping rank the resources with upvotes and I'll soon add more filtering options.

Would it be ok to link it on the README file, so users of the repo could also find it?

Thanks!

**Comment by paw31** [comment]
Hey @marcelscruz,
I checked out your project and it's really awesome.
But as you can see public-apis are entirely focused on APIs and Dev Resources is not an API but a website with other resources as well we cannot include its link in the README.

Wishing you all the best for Dev Resources.

**Comment by marcelscruz** [comment]
Hi @pawelbr, thank you for answering.

Maybe I expressed myself badly, but what I suggested was to, somehow, point the website as a "more visual" way of accessing the public APIs list, perhaps at the beginning of the README file, the same way as the public API for the project is also linked there.

Regardless of your decision, I appreciate you taking the time for answering and maintaining the project.
